src/build/strings.ts

```diff
@@ -106,7 +106,7 @@ const strings = {
   'browser.perScriptSourcemaps.description':
     'Whether scripts are loaded individually with unique sourcemaps containing the basename of the source file. This can be set to optimize sourcemap handling when dealing with lots of small scripts. If set to "auto", we\'ll detect known cases where this is appropriate.',
   'browser.userDataDir.description':
-    'By default, the browser is launched with a separate user profile in a temp folder. Use this option to override it. Set to false to launch with your default user profile.',
+    'By default, the browser is launched with a separate user profile in a temp folder. Use this option to override it. Set to false to launch with your default user profile. If `userDataDir` is specified, VS Code can not launch a new browser instance in the default profile if one is already running.',
```

Comment:
This phrasing is confusing, it'd be better to say "A new browser can't be launched if an instance is already running from the `userDataDir`."